README.md

```diff
@@ -89,6 +89,25 @@ Follow these steps to install the `chasten` program:
 - Type `pipx list` and confirm that Chasten is installed
 - Type `chasten --help` to learn how to use the tool
 
+## 🐋 Docker
+
+There is also the option to use [Docker](https://www.docker.com/) to interact with `chasten`
+
+Follow these steps to utilize Docker:
+
+- Install Docker Desktop for your operating system
```

Comment:
Should we provide a link to Docker Desktop? Maybe this link?

https://docs.docker.com/get-docker/